[fix][docs] Remove old template inlined (#20208)
apache/pulsar: wiki/proposals/PIP.md

> ## Template for a PIP design doc
> Located at https://github.com/apache/pulsar/blob/master/.github/ISSUE_TEMPLATE/pip.md

Reviewer: ```suggestion
## Template for a PIP design doc

Read [the template file](/.github/ISSUE_TEMPLATE/pip.md).
```
